sdk/identity/azure-identity/src/private/managed_identity_source.hpp

```diff
@@ -176,6 +176,7 @@ namespace Azure { namespace Identity { namespace _detail {
 
     explicit ImdsManagedIdentitySource(
         std::string const& clientId,
+        Azure::Core::Url const& imdsUrl,
```

Comment:
```suggestion
        Core::Url const& imdsUrl,
```
nit: can be one namespace shorter, similar to Core::Credentials in the next line